Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,278 @@
+                graphicsOverlay.addGraphics(associationsGraphics)
+                graphicsOverlay.addGraphic(boundingBoxGraphic)
+            }
+        }
+        
+        /// Creates content elements within the container element.
+        /// - Parameter containerElement: The container element.
+        /// - Returns: An array of utility elements in the container.
+        private func makeContentElements(containerElement: UtilityElement) async throws -> [UtilityElement] {
+            // Gets the containment associations from the element to display its content.
+            let containmentAssociations = try await network.associations(for: containerElement, ofKind: .containment)
+            // Determines the relationship of each element and add it to the content elements.
+            let contentElements: [UtilityElement] = containmentAssociations.map { association in
+                if association.fromElement.objectID == containerElement.objectID {
+                    return association.toElement
+                } else {
+                    return association.fromElement
+                }
+            }
+            return contentElements
+        }
+        
+        /// Creates the graphics for the utility elements within a container.
+        /// - Parameters:
+        ///   - contentElements: The elements within the container.
+        ///   - containerElement: The container element.
+        /// - Returns: An array of graphics.
+        private func makeGraphicsForContentElements(_ contentElements: [UtilityElement], within containerElement: UtilityElement)  async throws -> [Graphic] {
+            let contentFeatures = try await network.features(for: contentElements)
+            let graphics: [Graphic] = contentFeatures.compactMap { feature in
+                guard let featureTable = feature.table as? ServiceFeatureTable,
+                      let symbol = featureTable.layerInfo?.drawingInfo?.renderer?.symbol(for: feature) else {
+                    return nil
+                }
+                return Graphic(geometry: feature.geometry, symbol: symbol)
+            }
+            return graphics
+        }
+        
+        /// Creates the associations graphics for elements in the container.
+        /// - Parameter boundingExtent: The boundary for the container.
+        /// - Returns: An array of association relationship graphics.
+        private func makeGraphicsForAssociations(within boundingExtent: Envelope) async throws -> [Graphic] {
+            let containmentAssociations = try await network.associations(forExtent: boundingExtent)
+            let graphics: [Graphic] = containmentAssociations.map { association in
+                let symbol = association.kind == .attachment ? attachmentSymbol : connectivitySymbol
+                return Graphic(geometry: association.geometry, symbol: symbol)
+            }
+            return graphics
```

Comment:
If you want to:

```suggestion
            return try await network
                .associations(forExtent: boundingExtent)
                .map { association in
                    let symbol = association.kind == .attachment ? attachmentSymbol : connectivitySymbol
                    return Graphic(geometry: association.geometry, symbol: symbol)
                }
```

Reply:
I'll leave this and the one above unchanged to showcase each step with a separate local variable. We can link to a line of code more specifically in case anyone ask about the steps to create utility associations. 😛